patches/esm+3.2.25.patch

```diff
@@ -1,9 +1,9 @@
 diff --git a/node_modules/esm/esm/loader.js b/node_modules/esm/esm/loader.js
-index e0bbca5..8710025 100644
+index e0bbca5..0b5bfe8 100644
```

Comment:
I'm surprised this patch changed, since we didn't change the version of `esm` that we're using. Is this intentional? Same for the other patch.

Reply:
> I'm surprised this patch changed, since we didn't change the version of `esm` that we're using. Is this intentional?

Yes, as discussed in https://github.com/Agoric/agoric-sdk/issues/527#issuecomment-841747122

> Same for the other patch.

`nyc` is no longer referenced, so I needed to remove that patch or `patch-package` complains about its absence.